### PR TITLE
DOC more docs and example

### PR DESCRIPTION
**Pull request by beckermr**
I added more documentation for local integration testing and an example Dockerfile.

### CODE REVIEW

**Comment by beckermr** [comment]
@salilgupta1 This should be a quick one.

File: example/run.sh [hunk 1]
```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+echo "PLATFORM_OBJECT_ID=1089
```
[the remark this answers is not in the document]
Comment (beckermr) [reply example/run.sh]: Whoops!

**Comment by beckermr** [comment]
@salilgupta1 This is ready for a second look!

File: example/README.md [hunk 2]
```diff
@@ -13,8 +13,9 @@ In order to run the example, you need to have:
 
 To run the example:
 
-1. Clone this repo and then change to this directory: `cd civis-jupyter-notebook/example`
-2. Build the image using [build.sh](build.sh): `./build.sh`
+1. Clone this repo and then change to this directory: `cd civis-jupyter-notebook/example`.
+2. Build the image using [build.sh](build.sh): `./build.sh`.
 3. Change `<NOTEBBOK ID>` in [run.sh](run.sh) to the ID of your notebook from step 3 above.
```
Comment (salilgupta1) [inline example/README.md]: Typo should be `<NOTEBOOK ID>`

File: example/README.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+## Usage
+
+To run the example:
+
+1. Clone this repo and then change to this directory: `cd civis-jupyter-notebook/example`.
+2. Build the image using [build.sh](build.sh): `./build.sh`.
+3. Change `<NOTEBBOK ID>` in [run.sh](run.sh) to the ID of your notebook from step 3 above.
+4. Run the Jupyter notebook server using [run.sh](run.sh): `./run.sh`.
+5. Connect to the notebook at 
+   [http://localhost:8888/notebooks/notebook.ipynb](http://localhost:8888/notebooks/notebook.ipynb).
```
Comment (salilgupta1) [inline example/README.md]: My only comment here: the url will vary depending on how you decide to run docker. i.e. if you use docker-machine you'll actually need to point the IP of the docker-machine VM. I don't know if that should be clear to the user or if we should be explicit about that.

Comment (beckermr) [reply example/README.md]: Yeah. I had trouble forming the URL. I will add notes.

**Comment by beckermr** [comment]
@salilgupta1 Sorry for the typos! This one should be good to go now!

**Pull request merged**